BenadrylAutomation/src/main/java/com/jnj/pageobjects/ListingPage.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * @author Zahin Ahad
+ * @date 4-March-24
+ */
+
+package com.jnj.pageobjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+
+import com.jnj.actions.Action;
+import com.jnj.base.BaseClass;
+
+public class ListingPage extends BaseClass{
+    Actions actions;
+
+    /**
+     * Constructor
+     */
+    public ListingPage() {
+        PageFactory.initElements(driver, this);
+        actions = new Actions(driver);
+    }
+
+    /**
+     * Locators 
+     */
+    @FindBy(css = "#tab-menu-8651")
+    WebElement prodHeaderMenu;
+    
+    @FindBy(css = "#file-926 img")
+    WebElement bannerImage;
+    
+    @FindBy(css = ".inner > #content-main")
+    WebElement bannerHeading;
+
+    @FindBy(css = ".inner p")
+    WebElement bannerSubHeading;
+
+    @FindBy(css = "[data-value='56']")
+    WebElement adultOralQuickFilter;
+
+    @FindBy(css = "[data-value='711']")
+    WebElement adultTopicalQuickFilter;
+
+    @FindBy(css = "[data-value='61']")
+    WebElement kidsProdQuickFilter;
+	
+	@FindBy(xpath = "//div[@id='onetrust-close-btn-container']")
```

Comment:
These locators are not aligned properly. Please format the source code
@zahinSJI

Reply:
Formatting done

Reply:
Verified